stories/PopoverStory.re

```diff
@@ -0,0 +1,40 @@
+module PopoverTest1 = {
+  module Styles = {
+    open Css;
+
+    let popover =
+      style([
+        backgroundColor(hex("fff")),
```

Comment:
hmm this should come from the theme also

Reply:
won't we also keep the styles set in the component itself for ease usage?

Reply:
sure, I'm just testing here